### PR TITLE
Working izettle authorization call

### PR DESCRIPTION
**Pull request by Kivike**

### CODE REVIEW

File: app/MiekkaIntegration/Service/Izettle/Command/Authorization/Client.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+namespace MiekkaIntegration\Service\Izettle\Command\Authorization;
+
+use MiekkaIntegration\Service\Base\ClientInterface;
+use MiekkaIntegration\Service\DateTimeIso;
+use MiekkaIntegration\Service\Izettle\HttpClient;
+
+class Client implements ClientInterface
+{
+
+    CONST URL = 'https://oauth.izettle.com/token';
```
Comment (Clpsplug) [inline app/MiekkaIntegration/Service/Izettle/Command/Authorization/Client.php]: `CONST` -> `const`

File: app/MiekkaIntegration/Service/Izettle/Config.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+namespace MiekkaIntegration\Service\Izettle;
+
+use MiekkaIntegration\Config\ConfigProvider;
+
+class Config
+{
+
+    const KEY_USERNAME = 'IZETTLE_USERNAME';
+    const KEY_CLIENT_ID = 'IZETTLE_CLIENT_ID';
+    CONST KEY_SECRET = 'IZETTLE_CLIENT_SECRET';
```
Comment (Clpsplug) [inline app/MiekkaIntegration/Service/Izettle/Config.php]: `CONST` -> `const`

File: app/MiekkaIntegration/Service/Izettle/Config.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+    /**
+     * @return string
+     */
+    public function getClientSecret()
+    {
+        return $this->configProvider->getConfigValue(self::KEY_SECRET);
+    }
+
+    public function getPassword()
```
Comment (Clpsplug) [inline app/MiekkaIntegration/Service/Izettle/Config.php]: doc comment missing

File: app/MiekkaIntegration/Service/Izettle/Config.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+namespace MiekkaIntegration\Service\Izettle;
+
+use MiekkaIntegration\Config\ConfigProvider;
+
+class Config
+{
+
+    const KEY_USERNAME = 'IZETTLE_USERNAME';
+    const KEY_CLIENT_ID = 'IZETTLE_CLIENTID';
+    CONST KEY_SECRET = 'IZETTLE_SECRET';
```
Comment (jpaivarinta) [inline app/MiekkaIntegration/Service/Izettle/Config.php]: CONST -> const,  does it matter?

Comment (Clpsplug) [reply app/MiekkaIntegration/Service/Izettle/Config.php]: PHP forgives them, but ~my tommy gun~ [PSR-2 don't.](https://www.php-fig.org/psr/psr-2/#25-keywords-and-truefalsenull)

Comment (jpaivarinta) [reply app/MiekkaIntegration/Service/Izettle/Config.php]: ok :D

**Pull request merged**